src/js/tasks-function.js

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable max-classes-per-file */
+
```

Comment:
- [ ] Like the previous reviewer requested, kindly remove all `Eslint` disabling comments. Eslint helps us to write clean, DRY and scalable codes. Using an `Eslint` disabling comment defeats this purpose. Kindly remove it. 💻

Reply:
but if i remove this comment, I have to use just one class in this file "/* eslint-disable max-classes-per-file */"

Reply:
File has too many classes (3). Maximum allowed is 1

Reply:
Hi @naji-97, you can put the other classes in different modules and import them into the files. 😄